.env.example

```diff
@@ -1,13 +1,13 @@
 PROD=false
-NEXT_PUBLIC_INFURA_ID=de84e9a996114d32a3db3ca9ce7940b7
-NEXT_PUBLIC_EP_API=https://graph.everipedia.org/graphql
-NEXT_PUBLIC_CHAIN_ID=5
-NEXT_PUBLIC_BLOCK_EXPLORER_BASE_URL=https://goerli.etherscan.io/
-NEXT_PUBLIC_ALCHEMY_API_KEY=dgF6IdB0eWQgVz2m5Vo_3ei49RqAGAw5
-NEXT_PUBLIC_ENS_RPC=https://eth-goerli.g.alchemy.com/v2/dgF6IdB0eWQgVz2m5Vo_3ei49RqAGAw5
-NEXT_PUBLIC_ALCHEMY_CHAIN=Goerli
-NEXT_PUBLIC_GOOGLE_ANALYTICS=G-LB7XL58HPD
-NEXT_PUBLIC_IQ_ADDRESS=0x0552D756a3E92Aa874EF60F61b7a29030373e869
-NEXT_PUBLIC_HIIQ_ADDRESS=0xc03bcacc5377b7cc6634537650a7a1d14711c1a3
-NEXT_PUBLIC_HIIQREWARDS_ADDRESS=0x36Cae8d96CBB53e139628e63E47ebe2B47a53f1f
+NEXT_PUBLIC_INFURA_ID=
+NEXT_PUBLIC_EP_API=
+NEXT_PUBLIC_CHAIN_ID=
+NEXT_PUBLIC_BLOCK_EXPLORER_BASE_URL=
+NEXT_PUBLIC_ALCHEMY_API_KEY=
+NEXT_PUBLIC_ENS_RPC=
+NEXT_PUBLIC_ALCHEMY_CHAIN=
+NEXT_PUBLIC_GOOGLE_ANALYTICS=
+NEXT_PUBLIC_IQ_ADDRESS=
+NEXT_PUBLIC_HIIQ_ADDRESS=
+NEXT_PUBLIC_HIIQREWARDS_ADDRESS=
```

Comment:
its a good practice to leave the non critical test env. there. The only ones that need to be not pushed are private keys and so on